README.md

```diff
@@ -287,3 +287,54 @@ You can use these features to support additional languages, protocols, and serve
 
 [idl]: https://thrift.apache.org/docs/idl
 [iodata]: http://elixir-lang.org/getting-started/io-and-the-file-system.html#iodata-and-chardata
+
+## Quick and dirty FAQ
+
+### Why is it faster than the Apache implementation?
+The Apache Thrift implementation uses C++ to write Erlang modules that describe Thrift data
+structures, then uses that description to turn your Thrift data into bytes. It consults this
+description every time Thrift data is serialized. This on-the-fly conversion costs CPU time.
+
+Additionally, the separation of concerns in Thrift prevent the Erlang VM from doing the best
+job that it can do during serialization.
+
+On the other hand, this implementation uses Elixir to write Elixir that's specific
+ to _your_ thrift data. This serialization logic is then compiled, and that compiled code
+is what converts your data into bytes. We've spent a lot of time making sure
+that the generated serialization code takes advantage of several of the optimizations
+that the Erlang VM provides.
+
+### Can the Apache implementation achieve similar performance?
+
+It could, but they'd have to write similar Elixir code to what we're writing,
+and do it in C++. They would also have to re-implement `quote` and `unquote`
+in C++. This would be ...challenging, but it's not impossible. It's worth noting
+that while this is theoretically possible, the code would be much less maintainable,
+and the Apache implementation  isn't by any means slow. In the above benchmark,
+you can see that it serializes 1000 user objects in roughly 12ms, and deserializes
+1000 users in 21ms. If you're happy with this level of performance, then the
+Apache code should be fine.
+
+
+### What tradeoffs have you made to get this performance?
```

Comment:
I wanted to talk about why we're different from the rest of the thrift implementations. Maybe I've missed the mark.

(and to be fair, no one has asked about anything, since this isn't released ;) )